kitsune/wiki/views.py

```diff
@@ -1527,7 +1528,10 @@ def get_fallback_locale(doc, request):
     #   * wiki fallbacks for that locale
 
     for locale in all_accepted_locales:
-        if locale in translated_locales:
+        if locale == doc.locale:
```

Comment:
Yeah. I accept it would not be logical. moreover, it will raise issue about GA tracking as we are considering the other non en-US locale. Most of the accepted language header have en-US, so showing en-US content thorough the fallback system will not be the thing we expected.
So therefore, better exclude it from the list. And your approach is correct